index.html

```diff
@@ -62,56 +75,85 @@
 
               <li>
                 <a href="#news" class="nav-links news-link">News</a>
-              </li>  
-              
+              </li>
+
               <li>
-                  <a href="#" class="nav-links active">AiT Summit</a>
+                <a href="#" class="nav-links active">AiT Summit</a>
               </li>
             </ul>
-                        
-              
           </div>
+        </nav>
+      </header>
+
+      <section class="landing-page">
+        <div class="landing-text">
+          <h3>"Hello! Sharing world"</h3>
+          <h1 class="title">TECH IN AFRICA GLOBAL SUMMIT 2022</h1>
```

Comment:
- [ ] The `h1` is having appearance issues, though it is present, the users do not see it. This may be happening because of the background property applied to it. Kindly fix it 🙏🏻.